content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -114,7 +114,7 @@ Now, we will see how the autoscaler reacts to increased load.
 We will start a container, and send an infinite loop of queries to the php-apache service (please run it in a different terminal):
 
 ```shell
-kubectl run -it --rm load-generator --image=busybox /bin/sh
+kubectl run -i --tty busybox --image=busybox --restart=Never
```

Comment:
Hi @matt-sm 

- the `--rm` option is useful; it cleans up once the reader stops the container in that Pod
- I'd like to keep the name `load-generator` - is that OK?

If you want to cut down on the number of lines to copy, this will also work
(combining `kubectl run` and the `wget` loop)
```bash
kubectl run -i --tty load-generator --rm --image=busybox --restart=Never -- /bin/sh -c "while sleep 0.01; do wget -q -O- http://php-apache; done"
```
(and then remove the advice about pressing Enter and then typing in `while true; do wget -q -O- http://php-apache; done`)